internal/plugin/plugin.go

```diff
@@ -134,7 +136,9 @@ func reportAction(id uint32, req *inHTTP.Request, resp *inHTTP.Response) *flatbu
 }
 
 func HTTPReqCall(buf []byte, conn net.Conn) (*flatbuffers.Builder, error) {
-	req := inHTTP.CreateRequest(buf)
+	ctx, cancel := context.WithTimeout(context.Background(), 56*time.Second)
+	defer cancel()
+	req := inHTTP.CreateRequest(buf).WithContext(ctx)
```

Comment:
Better to set the ctx inside `CreateRequest` instead of providing it as a method.
As we will reuse the Request, such a method will make reusing impossible.